raiden/utils/gas_reserve.py

```diff
@@ -6,11 +6,12 @@
     GAS_REQUIRED_FOR_CLOSE_CHANNEL,
     GAS_REQUIRED_FOR_SETTLE_CHANNEL,
     GAS_REQUIRED_FOR_UNLOCK_1_LOCKS,
+    MAXIMUM_PENDING_TRANSFERS,
 )
 from raiden.transfer import views
 
 GAS_REQUIRED_FOR_CHANNEL_LIFECYCLE_AFTER_SETTLE = (
-    GAS_REQUIRED_FOR_UNLOCK_1_LOCKS
+    GAS_REQUIRED_FOR_UNLOCK_1_LOCKS * MAXIMUM_PENDING_TRANSFERS
```

Comment:
This is incorrect as explained here https://github.com/raiden-network/raiden/pull/2031#discussion_r207908006

Reply:
Will take care of it as per the discussion